### ISSUE TITLE
training performance is not as good as your model

### ISSUE THREAD

**Issue by xueyingliu** [mrharicot/monodepth#72]
I used **monodepth_main.py** for training Kitti dataset, it cost about 6 hours on 2 GPU( NVIDIA 1080ti ) and I get the following disparity image:
![0000000000](https://user-images.githubusercontent.com/30819873/32822656-9648f6ee-ca14-11e7-9375-ed1ea059d761.jpg)
                                                             the test picture

![0000000000_disp](https://user-images.githubusercontent.com/30819873/32822673-a9bce71c-ca14-11e7-821b-5919e501f593.png)
                                                       predicted by my training model

![0000000000_disp1](https://user-images.githubusercontent.com/30819873/32822708-c947a9e6-ca14-11e7-932c-1ce2af1d0692.png)
                                                       predicted by your given model

I was confused about the result, can you tell me what's wrong with my model?

**Comment by mrharicot** [comment]
Hi,
Can you please detail which parameters/dataset you used and what is the pretrained model that you tried?

**Comment by xueyingliu** [comment]
@mrharicot thanks for your reply,  my command is: 
`python monodepth_main.py --mode train --model_name my_model --encoder resnet50 --data_path ~/waq/KITTI/ --filenames_file ~/waq/monodepth/utils/kitti_filenames/kitti_train_files.txt --num_gpus 2 --log_directory ~/log/`
and kitti_train_files.txt is writen the same as  yours, and my kitti is about 165G, because I missed some images when transform them from **.png** to **.jpg**, and  I did not finetune, and use **resnet50** net, train from begining.

**Comment by mrharicot** [comment]
Can you evaluate your model on a KITTI image? Does it look good?

**Comment by xueyingliu** [comment]
@mrharicot Evaluation on KITTI images is not good, either. I think maybe my epochs(I use the default 50) is too small, now I am training with a larger one(250). Wish the result come out good.

**Comment by mrharicot** [comment]
When using all the default parameters, it should look good early on, training for longer will not help in this case.
You mentionned that some of your images are png and some are jpg, did you recreate a kitti_train_files.txt? If so did you shuffle the names?

**Comment by EthanZhangYi** [comment]
@xueyingliu  Hi, I am interested in the dataset you used above. It seems that it is a synthetic driving data. Could you please share some information about it?

**Comment by mrharicot** [comment]
closing this issue for now as @xueyingliu isn't responding

**Comment by vineetkr137** [comment]
How much approx time it will take on nvidia 1080 ti for a single frame

**Comment by xueyingliu** [comment]
@vineetkr137 I remember it takes about 1s for 10 frames, not very sure cause I am not on this project.

**Comment by xueyingliu** [comment]
@EthanZhangYi Hi, The data is from a game namd gtav, We simulated the stereo camera on a car and take driving pictures.

**Comment by vineetkr137** [comment]
@xueyingliu  thanks for your reply. now I am getting around 2.6~3.0 fps on jetson  tx2 and and 20~23 fps on 1080 Ti. is any thing more could be done (to optimize) for getting higher frame rate.

**Comment by mrharicot** [comment]
Hi,
The current code isn't very fast and wasn't optimised for speed.
The main modifications would be to make the model much smaller, such as using resnet18 for the encoder for instance.

**Comment by vineetkr137** [comment]
Hi, 
Thanks for your reply, can you please help me in detail what other things i need to download or edits to use encoder resnet50. because right now using --encoder resnet50 is showing errors.

**Comment by vineetkr137** [comment]
Hi @mrharicot 
Thanks for your reply, can you please help me in detail what other things i need to download or edits to use encoder resnet50. because right now using --encoder resnet50 and it is showing errors.
